bqemulatormanager/schema.py

```diff
@@ -1,48 +1,75 @@
+from __future__ import annotations
+
 import os
-from typing import List
```

Comment:
@kitagry [fyi] neither `Dict` nor `List` are required to import, you can use `list` `dict` (linter....)

Reply:
@yokomotod  Python3.8 cannot define the type.

> TypeError: 'type' object is not subscriptable

https://github.com/m3dev/bqemulatormanager/actions/runs/5286103390/jobs/9565213830

So, we need to import List and DIct

Reply:
I defined 3 types.

```python
TableSchema = List[Dict[str, Any]]
DatasetSchema = Dict[str, TableSchema]
ProjectSchema = Dict[str, DatasetSchema]
```